shimmingtoolbox/coils/create_coil_profiles.py

```diff
@@ -60,3 +59,89 @@ def create_coil_profiles(fnames_fmaps, list_currents):
         # static_offset = reg.intercept_.reshape(fmaps.shape[:-1])
 
     return profiles
+
+
+def generate_coil_bfield(wire, XYZ, gridsize):
```

Comment:
Maybe have a look at shimming toolbox/coils/biot_savart and see if It fits in that file better